ansible/server-state-playbooks/nightshade-web/playbook.yml

```diff
@@ -149,18 +151,6 @@
         remote_src: True
         creates: "{{ omero_web_extensionsdir }}/figure-{{omero_web_extensions_figure_ver}}" 
 
-    # download tagging
-    - name: OMERO.web apps | download latest tagging
-      become: yes
-      unarchive:
-        src: "http://downloads.openmicroscopy.org/webtagging/{{ omero_web_extensions_tagging_ver }}/webtagging-{{ omero_web_extensions_tagging_ver }}.zip"
-        owner: "root" 
-        group: "root" 
-        mode: "u=rwx,go=rx" 
-        dest: "{{ omero_web_extensionsdir }}"
-        remote_src: True
-        creates: "{{ omero_web_extensionsdir }}/webtagging-{{omero_web_extensions_tagging_ver}}" 
-
```

Comment:
Yep, thanks. It does remove the relevant lines from the OMERO.web config though, and restarts web/re-creates the config, which makes it take effect.